packages/viewer/src/scenes/scene.ts

```diff
@@ -3,6 +3,58 @@ import { Frame } from '../types';
 import { Camera } from './camera';
 import { Dimensions } from '@vertexvis/geometry';
 import { Raycaster } from './raycaster';
+import {
+  SceneItemOperations,
+  SceneOperationBuilder,
+  SelectorBuilder,
+  ItemSelectorBuilder,
+  SceneItemQuery,
+} from './operations';
+
+export class SceneItemOperationsExecutor
+  implements SceneItemOperations<SceneItemOperationsExecutor> {
+  protected builder = new SceneOperationBuilder();
+  private queryExecutor: SceneItemQueryExecutor;
+  public constructor(
+    queryExecutor?: SceneItemQueryExecutor,
```

Comment:
you can also do `queryExecutor = new SceneItemQueryExecutor()`, which is terser IMO.

Reply:
this piece was removed